File: Beecrowd/8-Geometria_Computacional/1039-Flores_de_Fogo-5/1039.cpp

#include <bits/stdc++.h>
using namespace std;

typedef long long ll;

int main(){
    double r1, x1, y1, r2, x2, y2, distance;
    while(cin>>r1>>x1>>y1>>r2>>x2>>y2){
        distance = sqrt((abs(x1-x2)*abs(x1-x2))+(abs(y1-y2)*abs(y1-y2)));
        if(distance+r2<=r1){
            cout<<"RICO\n";
        }
        else{
            cout<<"MORTO\n";
        }
    }
}

File: Beecrowd/8-Geometria_Computacional/1039-Flores_de_Fogo-5/1039.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        while (scanner.hasNext()) {
            double r1 = scanner.nextDouble();
            double x1 = scanner.nextDouble();
            double y1 = scanner.nextDouble();
            double r2 = scanner.nextDouble();
            double x2 = scanner.nextDouble();
            double y2 = scanner.nextDouble();

            double distance = Math.sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2));

            System.out.println(distance + r2 <= r1 ? "RICO" : "MORTO");
        }
        scanner.close();
    }
}
